core/jobs_test.py

```diff
@@ -657,6 +726,89 @@ def test_validity_of_each_continuous_computation_class(self):
                 for superclass in allowed_base_batch_job_classes]))
 
 
+class BaseMapReduceJobManagerForContinuousComputationsTests(
+        test_utils.GenericTestBase):
+
+    def test_raise_error_with_get_continuous_computation_class(self):
+        with self.assertRaisesRegexp(
+            NotImplementedError,
+            re.escape(
+                'Subclasses of BaseMapReduceJobManagerForContinuousComputations'
+                ' must implement the _get_continuous_computation_class() '
```

Comment:
Please place spaces at the end of strings, not at the beginning. Leave `BaseMapReduceJobManagerForContinuousComputations` on its own line if you have to.


Here and elsewhere.

Reply:
done